app/code/Magento/Newsletter/Controller/Manage.php

```diff
@@ -6,29 +6,53 @@
 namespace Magento\Newsletter\Controller;
 
 use Magento\Framework\App\RequestInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\UrlInterface;
 
 /**
  * Customers newsletter subscription controller
  */
 abstract class Manage extends \Magento\Framework\App\Action\Action
 {
+    /**
+     * Configuration path to newsletter active setting
+     */
+    const XML_PATH_NEWSLETTER_ACTIVE = 'newsletter/general/active';
+
     /**
      * Customer session
      *
      * @var \Magento\Customer\Model\Session
      */
     protected $_customerSession;
 
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $_scopeConfig;
```

Comment:
Can we make these properties private and remove underscore from names?

Reply:
All parameters in both classes.

Reply:
@melnikovi - I have updated the scope of variables. 
But i think we should use protected variables in parent classes so that these variables can be used in child classes. Else we need to re-declare the variable in child class as in this PR i had to re-declare the $customerSession in child class.
What do you think?

Reply:
Sorry, I meant fix only parameter names we introduced.

Reply:
@melnikovi - What should i do now? Do i need to restore the scope of variables to original scope.?
And do i need to restore the variable names to original names?